core/packages/contracts/contracts/DOTApp.sol

```diff
@@ -68,4 +68,13 @@ contract DOTApp is FeeController, AccessControl {
     {
         token.burn(feePayer, _amount);
     }
+
+    function transferVaultOwnership(
```

Comment:
Lets rather implement this after the XCMv3 work as we're likely going to have rewrite a lot of the app code.